external/wakaama/examples/shared/Make.defs

```diff
@@ -38,5 +38,9 @@
 CSRCS += commandline.c connection.c platform.c memtrace.c
 CFLAGS+=-I$(TOPDIR)/../external/wakaama/core
 
+ifeq ($(CONFIG_LWM2M_SECURITY),y)
```

Comment:
Could you please reply to a related question in commit: https://github.com/Samsung/TizenRT/pull/160? Thanks.